esp-hal/src/gpio/mod.rs

```diff
@@ -91,6 +91,7 @@ pub mod rtc_io;
 
 /// Convenience constant for `Option::None` pin
 
+#[ram]
```

Comment:
No way this isn't already in RAM. 🤔

Reply:
At some point it looked in the disassembly like there was a pointer in flash pointing to this ... but now it doesn't look like that from the linker-map anymore ... needs double checking

Reply:
I can understand that what this points to may live in flash, but the innards are an atomicptr so it's impossible to live in flash. Unless the compiler deduced that it never changes?

Reply:
I meant I think I have seen something in flash pointing to the instance in RAM but anyways need double checking

Reply:
double checked the disassembly .... it's not the case (and I agree it shouldn't) - so this is unnecessary